src/aiida/cmdline/commands/cmd_archive.py

```diff
@@ -132,6 +132,7 @@ def inspect(ctx, archive, version, meta_data, database):
     '-b', '--batch-size', default=1000, type=int, help='Stream database rows in batches, to reduce memory usage.'
 )
 @click.option('--test-run', is_flag=True, help='Determine entities to export, but do not create the archive.')
```

Comment:
Should we add the deprecation notice to the option description?

Reply:
Sure, why not